### PR TITLE
Increment version to 5.4.0

### LINKED ISSUE #8500: Branch release-5.3 and increment dev branch version to 5.4

### PR DESCRIPTION
**Pull request by donnie-msft**
Fixes: https://github.com/NuGet/Home/issues/8500

### CODE REVIEW

**Review by nkolev92 — COMMENTED** [review]
when changing any of the NuGetVersion props below, also update ProductVersion in src\NuGet.Clients\NuGet.Tools\NuGetPackage.cs

**Comment by donnie-msft** [comment]
> when changing any of the NuGetVersion props below, also update ProductVersion in src\NuGet.Clients\NuGet.Tools\NuGetPackage.cs

Done

File: build/config.props [hunk 1]
```diff
@@ -13,18 +13,18 @@
...
     <MajorNuGetVersion Condition="'$(MajorNuGetVersion)' == ''">5</MajorNuGetVersion>
-    <MinorNuGetVersion Condition="'$(MinorNuGetVersion)' == ''">3</MinorNuGetVersion>
+    <MinorNuGetVersion Condition="'$(MinorNuGetVersion)' == ''">4</MinorNuGetVersion>
     <PatchNuGetVersion Condition="'$(PatchNuGetVersion)' == ''">0</PatchNuGetVersion>
     <SemanticVersion Condition=" '$(SemanticVersion)' == '' ">$(MajorNuGetVersion).$(MinorNuGetVersion).$(PatchNuGetVersion)</SemanticVersion>
     
     <!-- ** Change for each new preview/rtm -->
     <!-- Check the VS schedule and manually enter a preview number here that makes sense. -->
-    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">rtm</ReleaseLabel>
+    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">preview1</ReleaseLabel>
```
Comment (zivkan) [inline build/config.props]: this is not SemVer2 compliant. Although I think this is how we've done it in the past, although 5.3 is the only version we did it "properly" :)

Comment (nkolev92) [inline build/config.props]: preview.1. 

We followed that in 5.3.x 

See: https://dotnet.myget.org/feed/nuget-build/package/nuget/NuGet.Protocol, 

https://dotnet.myget.org/feed/nuget-build/package/nuget/NuGet.Protocol/5.3.0-preview.2.6157+1917823cbfce7c33e23a7e37a2b64e66d25e8957

**Review by zivkan — APPROVED** [review]

File: build/config.props (hunk 1 above)
Comment (donnie-msft) [reply build/config.props]: Comment on there said it was weird and different than our pattern. Discussed just now and sounds like we do want to change the pattern, so I'll do that.

Comment (donnie-msft) [reply build/config.props]: Comment on there said it was weird and different than our pattern. Discussed just now and sounds like we do want to change the pattern, so I'll do that.

**Review by nkolev92 — APPROVED** [review]

**Pull request merged**